pkgs/development/tools/rust/cargo-cache/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, lib, fetchFromGitHub, rustPlatform, libiconv }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-cache";
+  version = "0.3.4";
+
+  src = fetchFromGitHub {
+    owner = "matthiaskrgr";
+    repo = pname;
+    # TODO: use v${version} when a tag is available
+    rev = "f3d113cc74644c4581c127e16b03e319677d669f";
+    sha256 = "1bc9c1r964w0766k3yhf79d30ldv37vyiji8wnqspk5sl2gzn0iq";
+  };
+
+  cargoSha256 = "1wzl8xvn3i5qgc4zmdqdh4kd8ww2ll2g14w4rf31i9lvq1ipa031";
+
+  buildInputs = lib.optional stdenv.isDarwin libiconv;
+
+  checkPhase = ''
+    cargo test -- \
+      --skip CargoCachePaths \
+      --skip alternative_registry_works \
+      --skip cargo_new_and_run_local
```

Comment:
After cursory look at these tests, my impression is that `alternative_registry_works` requires internet access, and the other two could probably be made to work by setting `HOME = "."`?

Reply:
Hi, yes, a couple of the cargo-cache tests require internet because I download crates/registry indices or initialize a new `$CARGO_HOME` at test-runtime.